internal/boxcli/create.go

```diff
@@ -63,7 +76,7 @@ func runCreateCmd(cmd *cobra.Command, args []string, flags *createCmdFlags) erro
 		path = filepath.Join(wd, flags.template)
```

Comment:
if `flags.template` is not set, the second part of filepath.Join() should be:

* filepath.Base(subdir) if subdir is set.
* filepath.Base(repo) otherwise.